test/ActiveRecordFindTest.php

```diff
@@ -19,38 +19,37 @@ public function test_find_with_no_params()
     }
 
     public function test_find_returns_single_model() {
-        $author = Author::find(3);
-        $this->assertInstanceOf(Model::class, $author );
+        $author = Author::find(3, ['select' => 'author_id']);
+        $this->assertInstanceOf(Model::class, $author);
 
         $author = Author::find('3');
-        $this->assertInstanceOf(Model::class, $author );
+        $this->assertInstanceOf(Model::class, $author);
```

Comment:
No objections to the style nit, or anything, but if they bother you, why not include the `test` folder in the `.php-cs-fixer.php` file?

Reply:
Implemented